README.md

```diff
@@ -17,9 +17,9 @@ We try to have an extensive documentation at [our wiki here at Github][4]:
 
 * [User Documentation][5]
 * [Contributor Documentation][6]
-  * [Volunteers Welcome!][7]
+* [Volunteers Welcome!][7]
 * [Developer Documentation][8]
-  * [Libraries Used][9]
+* [Libraries Used][9]
```

Comment:
These links were indented on purpose. They rightly belong to the respective sections. So, kindly revert these indentation changes.